content/applications/hr/payroll/work_entries.rst

```diff
@@ -2,76 +2,92 @@
+  dates and times for the work entry. First, click on either the :guilabel:`From` or :guilabel:`To`
+  line and a calendar pop-up appears. Select the date by navigating to the correct month and year
+  using the :guilabel:`< (left arrow)` and :guilabel:`> (right arrow)` icons, then click on the
+  specific day.
+
+  Next, select the time by clicking on either the hour or minute fields at the bottom of the
+  calendar, and select the desired time for both the hour and minutes.
+
+  When the date and time are correct for the entry, click the :guilabel:`Apply` button.
+- :guilabel:`Duration`: displays the hours based on the :guilabel:`To` and :guilabel:`From` entries.
+  Modifying this field modifies the :guilabel:`To` field (the :guilabel:`From` field does not
+  change).
 
-Enter the :guilabel:`Work Entry Name`, such as `Sick Time` or any other short description. Select
-the :guilabel:`Employee` and the :guilabel:`Work Entry Type` from the respective drop-down lists.
+Once the information is entered, click :guilabel:`Save & Close` to save the entry and close the
+pop-up.
 
 .. image:: work_entries/create.png
    :align: center
    :alt: Filling in the work entry Create form in Odoo.
 
-Next, enter the date and time for the work entry in the :guilabel:`From` and :guilabel:`To`
-drop-downs. First, select the date by navigating to the correct month and year using the
-:guilabel:`⬅️ (left arrow)` and :guilabel:`➡️ (right arrow)` icons, then click on the specific day.
-
-Then, select the time by clicking on the :guilabel:`⏰ (clock)` icon, and using the :guilabel:`⬆️ (up
-arrow)` and :guilabel:`⬇️ (down arrow)` icons for each section to enter the hour, minute, and second
-for the time period.
-
-The :guilabel:`Period` will display the hours based on the :guilabel:`To` and :guilabel:`From`
-entries.
-
-.. tip::
-   Before clicking either :guilabel:`Save & Close` or :guilabel:`Save & New`, it is good to double
-   check the :guilabel:`Period` to ensure the time stated corresponds to the :guilabel:`To` and
-   :guilabel:`From` fields.
-
-   .. image:: work_entries/period.png
-      :align: center
-      :alt: Hours entered in the Period field.
-
-Once the information is entered, click :guilabel:`Save & Close` to save the entry and close the
-pop-up, or :guilabel:`Save & New` to save the entry and create another :guilabel:`Work Entry Type`.
+.. _payroll/regenerate-work-entries:
 
 Regenerate work entries
 =======================
 
-After a work entry has been either added or modified, the work entries need to be regenerated for
-the affected employee(s). Click on the :guilabel:`Regenerate Work Entries` button at the top of the
-main dashboard, and a pop-up will appear.
+After work entries have been either added or modified, the work entries need to be regenerated for
+the affected employees. Click on the :guilabel:`Regenerate Work Entries` button at the top of the
```

Comment:
Lol- good point! First, I agree with using "they" - I just wanted to be 110% clear, which is why I used "work entries" again. Second-  I had checked with the HR experts, and they need to be regenerated when they have been changed or added, because (I think, but will double check) they were already generated once. When they were modified or added after the fact, they must be regenerated so they are computed correctly for the payslips.

So if I have a work entry that said I was working 8 hours that day but I actually only worked 4 (maybe because of sick time, vacation time, or just a regular old no-show) then the entry has to be updated to reflect what happened. Since they were already 'generated' they need to be 'regenerated' so they are updated and correct before issuing payslips.

I will re-check with the experts to make sure I am understanding this correctly, and I will expand on this so it is understood why we do this. Ill be honest, I have a 90% grasp on this but the last 10% is still "magic" to me =D

Reply:
OK, I was completely WRONG on how regenerating work entries works!! I chatted with Emilio (who has been helping me with HR questions) and he explained this for me. I have now updated this section, moved it (it didn't make sense to keep it here) and I also added a sentence in the very beginning of the doc, so this section makes sense. 

TL/DR: If conflicts happen form an app (time off, planning, etc) you can/should resolve the conflict in the APP that created the issue. THEN, you regenerate the work entries in payroll, which deletes the old entries and pulls new ones for the respective apps.

This keeps the data across ALL apps correct instead of just fixing them in the payroll app. Because if you DON'T fix it, then if you regenerate work entries- the issue appears again.